use-location.js

```diff
@@ -13,7 +13,7 @@ export default () => {
     return () => events.map(e => removeEventListener(e, handler));
   }, []);
 
-  return [path, to => history.pushState(0, 0, to)];
+  return [path, (to, replace) => history[replace && "replaceState" || "pushState"](0, 0, to)];
```

Comment:
The expression `replace && "replaceState" || "pushState"` was a bit hard to read. Can we use a ternary operator here maybe? Thanks!

Reply:
Yes, in the first version I did...

Reply:
Ok, so what made you change that?

Reply:
I don't know 🤦‍♂️

Reply:
Done